support-frontend/assets/components/checkoutExpander/checkoutExpander.tsx

```diff
@@ -24,20 +24,23 @@ const copyStyleLabel = css`
 const copyStyle = css`
 	${textSans.medium()};
 `;
+
 // ----- Types ----- //
 type ExpanderPropTypes = {
 	copy: string;
-	children: Node;
+	children: ReactNode;
 };
 
 // ----- Component ----- //
-const CheckoutExpander = ({ copy, children }: ExpanderPropTypes) => (
-	<details css={expander}>
-		<summary css={summary}>
-			<strong css={copyStyleLabel}>{copy}</strong>
-		</summary>
-		<div css={copyStyle}>{children}</div>
-	</details>
-); // ----- Exports ----- //
+function CheckoutExpander({ copy, children }: ExpanderPropTypes): JSX.Element {
+	return (
+		<details css={expander}>
+			<summary css={summary}>
+				<strong css={copyStyleLabel}>{copy}</strong>
+			</summary>
+			<div css={copyStyle}>{children}</div>
+		</details>
+	);
+} // ----- Exports ----- //
```

Comment:
Tiny thing, but maybe this should be on its own line? (Realise this is how it was before)

Reply:
Ah you're right, good catch. Will update now!